org.eclipse.emfcloud.validation/src/test/java/org/eclipse/emfcloud/validation/ValidationFrameworkTest.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************************
```

Comment:
/********************************************************************************
 * Copyright (c) 2019-2020 EclipseSource and others.
 *
 * This program and the accompanying materials are made available under the
 * terms of the Eclipse Public License v. 2.0 which is available at
 * https://www.eclipse.org/legal/epl-2.0, or the MIT License which is
 * available at https://opensource.org/licenses/MIT.
 *
 * SPDX-License-Identifier: EPL-2.0 OR MIT
 ********************************************************************************/